modules/gitops-enabling-the-application-set-resources-in-non-control-plane-namespaces.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * argocd_application_sets/managing-app-sets-in-non-control-plane-namespaces.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-enabling-the-application-set-resources-in-non-control-plane-namespaces_{context}"]
+= Enabling the application set resources in non-control plane namespaces
+
+As an Argo CD administrator, you can define a certain set of non-control plane namespaces wherein users can create, update, and reconcile `ApplicationSet` resources. You must explicitly enable and configure the `ArgoCD` and `ApplicationSet` custom resources (CRs) as per your requirements.
+
+.Procedure
+
+. Set the `sourceNamespaces` parameter for the `applicationSet` spec to include the non-control plane namespaces:
++
+.Example Argo CD custom resource
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: example-argocd
+spec:
+  applicationSet:
+    sourceNamespaces: <1>
+      - dev <2>
+----
+<1> List of non-control plane namespaces for creating and managing `ApplicationSet` resources
+<2> Name of the target namespace for the Argo CD server to create and manage `ApplicationSet` resources
++
+[NOTE]
+====
+At the moment, the use of wildcards (*) is not supported in the `.spec.applicationSet.sourceNamespaces` field.
+====
+
+. Verify that the following role-based access control (RBAC) resources are either created or modified by the {gitops-shortname} Operator:
++
+|===
+|**Name** |**Kind** |**Purpose**
+|`<argocdName-argocdNamespace>-argocd-applicationset-controller` |`ClusteRole` and `ClusterRoleBinding` |For the Argo CD ApplicationSet Controller to watch and list `ApplicationSet` resources at cluster-level.
```

Comment:
I also just noticed this `ClusteRole` typo, so make sure to get that also in whatever you end up doing:
```suggestion
|`<argocdName-argocdNamespace>-argocd-applicationset-controller` |`ClusterRole` and `ClusterRoleBinding` |For the Argo CD ApplicationSet Controller to watch and list `ApplicationSet` resources at cluster-level.
```

Reply:
thanks @adellape. We already have a [JIRA ticket ](https://issues.redhat.com/browse/RHDEVDOCS-5940)to focus on them. WIll address it as part of that issue.